app/SmartAlbums/Utils/MimicModel.php

```diff
@@ -25,11 +25,9 @@ abstract public function toArray(): array;
 	 *
 	 * @param string $key
 	 *
-	 * @return mixed
-	 *
 	 * @throws InternalLycheeException
 	 */
-	public function __get(string $key)
+	public function __get(mixed $key)
```

Comment:
Then you need to throw a `LycheeInvalidArgumentException()` if `$key` is not of type string.